src/pages/workspace/accounting/netsuite/import/NetSuiteImportCustomFieldEdit.tsx

```diff
@@ -0,0 +1,170 @@
+import React, {useCallback, useMemo} from 'react';
+import type {ValueOf} from 'type-fest';
+import ConnectionLayout from '@components/ConnectionLayout';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapper from '@components/Form/InputWrapper';
+import type {FormInputErrors, FormOnyxValues} from '@components/Form/types';
+import TextInput from '@components/TextInput';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {updateNetSuiteCustomLists, updateNetSuiteCustomSegments} from '@libs/actions/connections/NetSuiteCommands';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import * as PolicyUtils from '@libs/PolicyUtils';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import type {WithPolicyConnectionsProps} from '@pages/workspace/withPolicyConnections';
+import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type {NetSuiteCustomList, NetSuiteCustomSegment} from '@src/types/onyx/Policy';
+import NetSuiteCustomFieldMappingPicker from './NetSuiteImportCustomFieldNew/NetSuiteCustomFieldMappingPicker';
+
+type CustomRecord = NetSuiteCustomList | NetSuiteCustomSegment;
```

Comment:
We shouldn't call CustomLists "Custom Records" since they are distinct objects, and we don't want to inadvertently conflate the two for future developers. If anything both Custom Lists and Custom Records/Segments should be under the umbrella of "Custom Field", and then we're editing _properties_ of the "Custom Field"

Does that make sense?

Reply:
Okay makes sense.

Reply:
Renamed to `CustomField` and `customField` across.